src/NLU.DevOps.Core/LabeledUtteranceConverter.cs

```diff
@@ -97,6 +97,14 @@ public override Entity ReadJson(JsonReader reader, Type objectType, Entity exist
                     jsonObject.Remove("endPos");
                 }
 
+                var entity = jsonObject.Value<string>("entity");
+                var entityType = jsonObject.Value<string>("entityType");
+                if (entityType == null && entity != null)
+                {
+                    jsonObject.Add("entityType", entity);
```

Comment:
>entityType [](start = 36, length = 10)

why don't you completely remove entity and only support entityType?

Reply:
Because LUIS batch format uses "entity".

Reply:
I can add doc comments so this is more explicit.